cmd/operator/main.go

```diff
@@ -183,6 +184,10 @@ func run(fs *flag.FlagSet) int {
 		stdlog.Fatal(err)
 	}
 
+	if _, err := maxprocs.Set(); err != nil {
```

Comment:
We can provide logger to the maxprocs.set as well, for reference: https://github.com/prometheus/prometheus/blob/d699dc3c7706944aafa56682ede765398f925ef0/cmd/prometheus/main.go#L724

Reply:
Thanks. @nicolastakashi 
Fixed. Please re-check